wp/wp-content/themes/lahma_maker/template-parts/content-arcsi.php

```diff
@@ -67,8 +67,11 @@ function sortAssociativeArrayByKey($array, $key, $direction){
 <h3>Arcsived shows for <?php the_title(); ?></h3>
 
 <div class="sort-block">
+    <button id="alphabetical" style="margin-right: 1rem;">
+        <i class="fa fa-sort-alpha-asc" aria-hidden="true"></i> Order by Title
```

Comment:
Now that we have more than 1 button what do you think of highlighting the selected one? Now both are transparent and normal

Reply:
Nice highlighting!